Maddori.Apple/Maddori.Apple/Screen/Setup/SetNickname/SetNicknameViewController.swift

```diff
@@ -118,8 +119,10 @@ final class SetNicknameViewController: BaseViewController {
     
     // MARK: - life cycle
     
-    init(from: ViewType) {
+    init(from: ViewType, teamId: Int?, teamName: String) {
```

Comment:
p5;

CreateTeamViewController 에서 teamId = nil 이라고 하길래 그거보단 여기에서 
```suggestion
    init(from: ViewType, teamId: Int? = nil, teamName: String) {
```
이렇게 잡아주면 CreateTeamViewController 에서 teamId 를 작성하지 않아도 돼서 더 깔끔하지 않나 싶어요!

Reply:
[f7acc4b](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/359/commits/f7acc4b752e5af0e3d638edfd69575381de0c46f)
오이 감사합니다~ 이러면 nil을 안 써도 됐군용 ㅎㅎ